godotproject/project.godot

```diff
@@ -17,7 +17,6 @@ _global_script_class_icons={
 
 config/name="Frog Goes To Space - Cooper Prototype"
 run/main_scene="res://Menus/MainMenu/MainMenu.tscn"
-config/icon="res://icon.png"
```

Comment:
Oh I deleted the godot icon, I didnt realize it was in use here should I add it back in?

Reply:
If it gives errors without it, I'd say add it back in

Reply:
added back in, no more errors.